### PR TITLE
Fixes #28966 - Add pulp3 content switchover command

### PR DESCRIPTION
**Pull request by johnpmitsch**
Adds a command to switch content from pulp2 to 3

### CODE REVIEW

**Comment by johnpmitsch** [comment]
I'm not sure if I have to set something up on nightly, but this is what I'm seeing when running:

```
[root@centos7-katello-nightly foreman_maintain]# ./bin/foreman-maintain advanced procedure run content-switchover
Running ForemanMaintain::Scenario
================================================================================
Convert supported content from Pulp 2 to Pulp 3:                      [FAIL]
Failed executing foreman-rake katello:pulp3_migration, exit status 1:
 rake aborted!
ForemanTasks::TaskError: Task f2be27c2-5201-4719-adcf-76861967568a: Dynflow::Errors::UnknownError[Pulp2to3MigrationClient::ApiError]: Error message: the server returns an error
HTTP status code: 404
Response headers: {"date"=>"Mon, 10 Feb 2020 13:55:56 GMT", "server"=>"gunicorn/20.0.4", "content-type"=>"text/html; charset=UTF-8", "x-frame-options"=>"SAMEORIGIN", "content-length"=>"77", "via"=>"1.1 centos7-katello-nightly.jomitsch.example.com", "connection"=>"close"}
Response body: <h1>Not Found</h1><p>The requested resource was not found on this server.</p>
/opt/theforeman/tfm/root/usr/share/gems/gems/katello-3.15.0.pre.master/lib/katello/tasks/pulp3_migration.rake:13:in `block (2 levels) in <top (required)>'
/opt/rh/rh-ruby25/root/usr/share/gems/gems/rake-12.3.0/exe/rake:27:in `<top (required)>'
Tasks: TOP => katello:pulp3_migration
```

File: definitions/procedures/content/switchover.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module Procedures::Content
+  class Switchover < ForemanMaintain::Procedure
+    metadata do
+      description 'Convert supported content from Pulp 2 to Pulp 3'
```
Comment (jlsherrill) [inline definitions/procedures/content/switchover.rb]: I'd actually change this to 'Switch support for certain content from Pulp 2 to Pulp 3'

Comment (johnpmitsch) [reply definitions/procedures/content/switchover.rb]: @jlsherrill updated!

File: definitions/procedures/content/switchover.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+      for_feature :pulpcore
+    end
+
+    def run
+      execute!('foreman-rake katello:pulp3_migration')
+      execute!('foreman-rake katello:pulp3_post_migration_check')
+      execute!('foreman-rake katello:pulp3_content_switchover')
+      args = %( --foreman-proxy-content-proxy-pulp-isos-to-pulpcore=true
+                 --katello-use-pulp-2-for-file=false
+                 --katello-use-pulp-2-for-docker=false)
```
Comment (jlsherrill) [inline definitions/procedures/content/switchover.rb]: I'm getting: 
sh: line 2: --katello-use-pulp-2-for-docker=false: command not found

it seems like those all need to be on the same line, or probably use \  at the end?

Comment (johnpmitsch) [reply definitions/procedures/content/switchover.rb]: @jlsherrill thanks, I just moved it to an array of strings that is joined so its still easy to read and we don't have to deal with multi-line ruby strings and compatibility, indentation, + new lines.

**Pull request merged**